continuous_integration/travis/test_imports.sh

```diff
@@ -27,10 +27,10 @@ test_import () {
 conda create -n test-imports python=$PYTHON
 source activate test-imports
 
-(test_import "Core" "" "import dask, dask.threaded, dask.optimization") && \
-(test_import "Delayed" "toolz" "import dask.delayed") && \
-(test_import "Bag" "fsspec toolz partd cloudpickle" "import dask.bag") && \
-(test_import "Array" "toolz numpy" "import dask.array") && \
-(test_import "Dataframe" "fsspec numpy pandas toolz partd cloudpickle" "import dask.dataframe") && \
+(test_import "Core" "psutil" "import dask, dask.threaded, dask.optimization") && \
```

Comment:
Similar comment here about the `psutil` requirement. These import checks can be removed too